jnigen/lib/src/bindings/pure_dart_bindings.dart

```diff
@@ -218,24 +219,27 @@ class PureDartBindingsGenerator extends BindingsGenerator {
       'DO NOT EDIT!\n\n';
   static const defaultLintSuppressions =
       '// ignore_for_file: annotate_overrides\n'
+      '// ignore_for_file: camel_case_extensions\n'
       '// ignore_for_file: camel_case_types\n'
       '// ignore_for_file: constant_identifier_names\n'
       '// ignore_for_file: file_names\n'
       '// ignore_for_file: no_leading_underscores_for_local_identifiers\n'
       '// ignore_for_file: non_constant_identifier_names\n'
+      '// ignore_for_file: unnecessary_cast\n'
       '// ignore_for_file: unused_element\n'
       '// ignore_for_file: unused_field\n'
       '// ignore_for_file: unused_import\n'
       '// ignore_for_file: unused_shown_name\n'
       '\n';
 
+  static const ffiImport = 'import "dart:ffi" as ffi;\n';
   static const jniImport = 'import "package:jni/jni.dart" as jni;\n\n';
   static const internalHelpersImport =
       'import "package:jni/internal_helpers_for_jnigen.dart";\n\n';
-  static const defaultImports = jniImport + internalHelpersImport;
+  static const defaultImports = ffiImport + jniImport + internalHelpersImport;
```

Comment:
It is only used for `JniArray<primitives>`. I could generate `jni.JniArray<jni.JInt>` instead of `jni.JniArray<ffi.Int32>` and we wouldn't need to import ffi.